yt_dlp/extractor/locipo.py

```diff
@@ -0,0 +1,147 @@
+from .common import InfoExtractor
+from ..utils import (
+    clean_html,
+    int_or_none,
+    parse_iso8601,
+    parse_qs,
+    str_or_none,
+    url_or_none,
+)
+from ..utils.traversal import require, traverse_obj
+
+
+class LocipoBaseIE(InfoExtractor):
+    _BASE_URL = 'https://locipo.jp'
+
+    def _call_api(self, path, item_id):
+        return self._download_json(
+            f'https://api.locipo.jp/api/v1/{path}', item_id)
+
+
+class LocipoIE(LocipoBaseIE):
+    _VALID_URL = r'https?://locipo\.jp/(?:creative|embed)/(?:\?(?:[^#]+&)?id=)?(?P<id>[\da-f]{8}(?:-[\da-f]{4}){3}-[\da-f]{12})'
+    _TESTS = [{
+        'url': 'https://locipo.jp/creative/fb5ffeaa-398d-45ce-bb49-0e221b5f94f1',
+        'info_dict': {
+            'id': 'fb5ffeaa-398d-45ce-bb49-0e221b5f94f1',
+            'ext': 'mp4',
+            'title': 'リアルカレカノ#4 ～伊達さゆりと勉強しよっ？～',
+            'description': 'md5:70a40c202f3fb7946b61e55fa015094c',
+            'duration': 3622,
+            'genres': 'count:1',
+            'modified_date': '20240904',
+            'modified_timestamp': 1725415481,
+            'release_timestamp': 1711789200,
+            'release_date': '20240330',
+            'series': 'リアルカレカノ',
+            'series_id': 'b865b972-99fe-41d5-a72c-8ed5c42132bd',
+            'thumbnail': r're:https?://.+\.(?:jpg|png)',
+            'timestamp': 1711789200,
+            'upload_date': '20240330',
+            'uploader': '東海テレビ',
+            'uploader_id': 'thk',
+        },
+    }, {
+        'url': 'https://locipo.jp/embed/?id=71a334a0-2b25-406f-9d96-88f341f571c2',
+        'info_dict': {
+            'id': '71a334a0-2b25-406f-9d96-88f341f571c2',
+            'ext': 'mp4',
+            'title': '#1 オーディション／ゲスト伊藤美来、豊田萌絵',
+            'description': 'md5:5bbcf532474700439cf56ceb6a15630e',
+            'duration': 3399,
+            'modified_date': '20250704',
+            'modified_timestamp': 1751623614,
+            'release_timestamp': 1751623200,
+            'release_date': '20250704',
+            'series': '声優ラジオのウラカブリ～Locipo出張所～',
+            'series_id': 'eaf2f2b2-aa73-40f1-b4c9-e47f098775b8',
+            'thumbnail': r're:https?://.+\.(?:jpg|png)',
+            'timestamp': 1751623200,
+            'upload_date': '20250704',
+            'uploader': 'テレビ愛知',
+            'uploader_id': 'tva',
+        },
+    }, {
+        'url': 'https://locipo.jp/creative/860201fa-f22b-4ffd-8890-6320a857159f?list=fef7c4fb-741f-4d6a-a3a6-754f354302a2',
+        'info_dict': {
+            'id': 'fef7c4fb-741f-4d6a-a3a6-754f354302a2',
+            'title': 'CBCアナウンサー公式【みてちょてれび】',
+            'description': 'md5:50a1b23e63112d5c06c882835c8c1fb1',
+            'genres': 'count:1',
+            'modified_date': r're:\d{8}',
+            'modified_timestamp': int,
+            'thumbnail': r're:https?://.+\.(?:jpg|png)',
+        },
+        'playlist_mincount': 32,
+    }]
```

Comment:
please add a `--no-playlist` test
```suggestion
    }, {
        'url': 'https://locipo.jp/creative/a0751a7f-c7dd-4a10-a7f1-e12720bdf16c?list=006cff3f-ba74-42f0-b4fd-241486ebda2b',
        'info_dict': {
            'id': 'a0751a7f-c7dd-4a10-a7f1-e12720bdf16c',
            'ext': 'mp4',
            'title': '#839 人間真空パック',
            'cast': 'count:1',
            'description': 'md5:9fe190333b6975c5001c8c9cbe20d276',
            'duration': 180,
            'genres': 'count:3',
            'modified_date': '20250503',
            'modified_timestamp': 1746239489,
            'release_timestamp': 1746239100,
            'release_date': '20250503',
            'series': 'でんじろう先生のはぴエネ！',
            'series_id': '006cff3f-ba74-42f0-b4fd-241486ebda2b',
            'thumbnail': r're:https?://.+\.(?:jpg|png)',
            'timestamp': 1746239400,
            'upload_date': '20250503',
            'uploader': '中京テレビ',
            'uploader_id': 'ctv',
        },
        'params': {'noplaylist': True},
    }]
```